firmware/application/apps/ui_playlist.cpp

```diff
@@ -325,7 +326,8 @@ void PlaylistView::update_ui() {
         text_filename.set(current()->path.filename().string());
         text_sample_rate.set(unit_auto_scale(current()->metadata.sample_rate, 3, 0) + "Hz");
 
-        auto duration = ms_duration(current()->file_size, current()->metadata.sample_rate, 4);
+        bool c8 = path_iequal(current()->path.extension(), c8_ext);
+        auto duration = ms_duration(c8 ? current()->file_size * 2 : current()->file_size, current()->metadata.sample_rate, 4);
```

Comment:
nit: in my brain doubling the filesize doesn't make as much sense as halving the bytes/sample

`auto duration = ms_duration(current()->file_size, current()->metadata.sample_rate, c8 ? 2 : 4);`

Reply:
Fixed this b4 seeing your comment  :-)